src/controllers/controller.bar.js

```diff
@@ -112,10 +112,9 @@ module.exports = function(Chart) {
 			var me = this;
 			var meta = me.getMeta();
 			var yScale = me.getScaleForId(meta.yAxisID);
-			var base = yScale.getBaseValue();
-			var original = base;
+			var base = 0;
```

Comment:
Not sure but I would tend to think that `base` still need to be initialized to the scale base value because when not stacked this method returns `getBasePixel()` which is `getPixelForValue(getBaseValue())`.